### PR TITLE
[feat-2742]: Handle logging config

### PR DESCRIPTION
**Pull request by keon94**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [ ] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation & [PR Template](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [ ] This PR is using a `label` (bug, feature etc.)
- [ ] My code is has necessary documentation (if appropriate)
- [ ] I have added any relevant tests
- [ ] This section (**⚠️ &nbsp;&nbsp;Pre Checklist**) will be removed when submitting PR

# Summary

### Does this close any open issues?
#2742 

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by keon94** [comment]
@warren830 @hezyin could any of you guys look at and approve this?

**Comment by klesh** [comment]
Hi, @keon94 I suggest that we go with "no LOGGING_DIR, no logging files", using a **relative directory** would cause multiple dirs to be created

**Comment by keon94** [comment]
> Hi, @keon94 I suggest that we go with "no LOGGING_DIR, no logging files", using a **relative directory** would cause multiple dirs to be created

Hey Klesh, I've added LOGGING_DIR to docker-compose instead with volume mapping like we discussed earlier today.

**Comment by keon94** [comment]
Returning 404 if log file is not present now. FYI @e2corporation @klesh 

image

File: .env.example [hunk 1]
```diff
@@ -27,7 +27,6 @@ TEMPORAL_URL=
 TEMPORAL_TASK_QUEUE=
 # Debug Info Warn Error
 LOGGING_LEVEL=
-LOGGING_DIR=./logs
```
Comment (klesh) [inline .env.example:30]: We can keep this key, but let it be empty.

File: services/pipeline.go [hunk 2]
```diff
@@ -389,13 +389,13 @@ func CancelPipeline(pipelineId uint64) error {
...
-	path := filepath.Dir(pipelineLog.GetConfig().Path)
+	path := pipelineLog.GetConfig().Path
 	_, err := os.Stat(path)
 	if err == nil {
-		return path, nil
+		return filepath.Dir(path), nil
 	}
 	if os.IsNotExist(err) {
-		return "", fmt.Errorf("logs for pipeline #%d not found: %v", pipeline.ID, err)
+		return "", fmt.Errorf("logs for pipeline #%d not found: %w", pipeline.ID, err)
```
Comment (klesh) [inline services/pipeline.go:398]: Can we add more hint to the error message? like "you have to specify the LOGGING_DIR first"

**Comment by klesh** [comment]
Good job!! I like it overall. Please fix the minor issues, and we can merge and test.

File: services/pipeline.go (hunk 2 above)
Comment (keon94) [reply services/pipeline.go]: done

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**